api/src/test/java/org/openmrs/module/pihcore/ConceptDictionaryExportTest.java

```diff
@@ -40,7 +40,7 @@
  * mvn test -Dtest=ConceptDictionaryExportTest -Dconcept_export_properties_file=/tmp/concept_export.properties
  */
 @SkipBaseSetup
-@Ignore
+@Disabled
```

Comment:
The new annotation for ignore, I assume?

Reply:
Seems that way in Junit 5 yes